src/current/v23.2/physical-cluster-replication-overview.md

```diff
@@ -122,11 +122,15 @@ Statement | Action
 
 ### Cluster versions and upgrades
 
-The standby cluster must be at the same version as, or one version ahead of, the primary's application virtual cluster at the time of cutover.
+When PCR is enabled, we recommend upgrading with the following procedure. This upgrades the standby cluster before the primary cluster. It is preferable to avoid a situation where the virtual cluster, which is being replicated, is a version higher than what the standby cluster can serve if you were to cut over. Within the primary and standby CockroachDB clusters, the system virtual cluster must be at a cluster version greater than or equal to the virtual cluster:
 
-To [upgrade]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}) a virtualized cluster, you must carefully and manually apply the upgrade. For details, refer to [Upgrades]({% link {{ page.version.version }}/work-with-virtual-clusters.md %}#upgrade-a-cluster) in the [Cluster Virtualization Overview]({% link {{ page.version.version }}/cluster-virtualization-overview.md %}).
+1. [Upgrade the binaries]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#step-4-perform-the-rolling-upgrade) on the primary and standby clusters. Replace the binary on each node of the cluster and restart the node.
+1. [Finalize]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#step-6-finish-the-upgrade) the upgrade on the standby's system virtual cluster.
+1. [Finalize]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#step-6-finish-the-upgrade) the upgrade on the primary's system virtual cluster.
+1. [Finalize]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#step-6-finish-the-upgrade) the upgrade on the standby's virtual cluster.
+1. [Finalize]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#step-6-finish-the-upgrade) the upgrade on the primary's virtual cluster.
 
-When physical cluster replication is enabled, we recommend following this procedure on the standby cluster first, before upgrading the primary cluster. It is preferable to avoid a situation in which the application virtual cluster, which is being replicated, is a version higher than what the standby cluster can serve if you were to cut over.
+The standby cluster must be at the same version as, or one version ahead of, the primary's virtual cluster at the time of [cutover]({% link {{ page.version.version }}/cutover-replication.md %}).
```

Comment:
We should be more emphatic about this -- you absolutely should not start or do steady state or cutover in PCR when the standby cluster is at a lower version than the primary. I think we'll add guardrails for this soon but as of right now we should really call this out in the docs (maybe like a warning sign box)?

Reply:
I've added this to a warning box in this section and in the ## Start clusters section